content/addons.md

```diff
@@ -68,27 +68,35 @@ ember addon <addon-name> [options]
 
 The result is the creation of a directory called `<addon-name>`, which has many files and looks a bit like an Ember app. We won't need to use all the files to make a useful addon. By convention, _most_ Ember addons start with `ember` in the name, like `ember-basic-dropdown`. This will help other developers find our addon.
 
-<!-- Should we cover in-repo addons at all??? -->
-<!-- 
-If the addon is just meant to be used in a single project, an "in-repo" addon could be created instead. The benefit is that it is lightweight, but there are some major limitations; an in-repo addon can't be shared between apps, versioned independently, or published to npm. From within an existing Ember app, use:
+### Addon file structure
 
-```bash
-ember generate in-repo-addon <addon-name> [options]
-```
+In some ways, an addon is like a mini Ember app. It has a very similar file structure, uses a lot of the same API methods, and can do most things that components are able to do. 
 
-This generates a folder called `lib/<addon-name>` that contains its own `package.json` and an `index.js` file. 
--->
+Let's take a look a some of the most important files and folders in an addon, and how they are different from what we would find in an app. 
 
-### Addon file structure
+#### `addon/` 
 
-In some ways, an addon is like a mini Ember app. It has a very similar file structure, uses a lot of the same API methods, and can do most things that components are able to do. 
+If an addon only offers one UI template, the addon files usually go in the `addon/` directory. When the addon is used in a template, it is called by the "real name" of the addon, like `{{my-addon-name someValue=true}}`. This directory can hold many of the same subdirectories and files that an Ember app would, like `/components/` and `/templates/`.
```

Comment:
by 'UI template' do you mean a component template?

Reply:
to me first two sentences lead to misunderstanding. This special cases better put into separate sections

This directory can hold many of the same subdirectories and files that an Ember app would -> This directory has the same structure as Ember application `app` folder. For example `components/`, 'templates/' and so on

Reply:
Yeah, backing up what @kategengler said, I didn't even know the single-template things was possible, and it's basically never done.  Even if it does work, I would strongly recommend against documenting it in a beginners guide to addons.

Also "UI template" is confusing because that's not how we talk about Ember anywhere else. We tend to just say "template" or "component" depending on the context.